pkg/kubelet/cm/cgroup_manager_test.go

```diff
@@ -0,0 +1,94 @@
+// +build linux
```

Comment:
Change the filename based on the discussion in https://github.com/vishh/kubernetes/pull/2/files#r66893095